scripts/scripts.js

```diff
@@ -31,6 +32,31 @@ function buildHeroBlock(main) {
   }
 }
 
+function buildautoblogBlock(main) {
```

Comment:
We use camel case for functions, so please change  all to "buildAutoblogBlock"

Reply:
Done